specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2023-04-01-preview/common-types.json

```diff
@@ -179,6 +179,37 @@
       },
       "x-ms-azure-resource": true
     },
+    "IdentityMetadata": {
```

Comment:
>IdentityMetadata [](http://example.com/codeflow?start=5&length=16)

consider referring the type from the actual commontypes instead ? [https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/managedidentity.json#L50](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/managedidentity.json#L50)

Reply:
Updated.